### PR TITLE
socket_summary add more tcp status

### PR DESCRIPTION
**Pull request by yaule**

### CODE REVIEW

**Comment by elasticcla** [comment]
Hi @yaule, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in yout Git [commit](https://github.com/elastic/beats/pull/9430.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

**Comment by sayden** [comment]
Hi @yaule Thanks for the PR. We'll need also a CHANGELOG entry. CI error is probably because a `make update` command is missing.

**Comment by sayden** [comment]
Awesome! We are one step closer, now we need to add some test. Take a look at the Beat Developer Guide https://www.elastic.co/guide/en/beats/metricbeat/5.5/metricbeat-developer-guide.html and check any other module for examples, like MySQL

**Comment by sayden** [comment]
Here you can see the current test of the module that you can also modify to add the contents of this PR https://github.com/elastic/beats/blob/037e45f566228976f65b8b6352734f6aa921696e/metricbeat/module/system/socket_summary/socket_summary_test.go

**Comment by yaule** [comment]
hi, @sayden . I submitted the code for the first time. Now I don't know what to do. Please tell me what I should do now.

**Comment by jsoriano** [comment]
Hi @yaule, I think the code looks mostly fine, failing test in CI is not related.

**Comment by jsoriano** [comment]
jenkins, test this

File: metricbeat/module/system/socket_summary/_meta/fields.yml [hunk 1]
```diff
@@ -35,6 +35,18 @@
               type: integer
               description: >
                 All TCP listening ports
+            - name: established
+              type: integer
+              description: >
+                TCP established ports
```
Comment (jsoriano) [inline metricbeat/module/system/socket_summary/_meta/fields.yml]:
```suggestion
                Number of established TCP connections
```

File: metricbeat/module/system/socket_summary/_meta/fields.yml [hunk 2]
```diff
@@ -35,6 +35,18 @@
...
+            - name: close_wait
+              type: integer
+              description: >
+                TCP close_wait ports
```
Comment (jsoriano) [inline metricbeat/module/system/socket_summary/_meta/fields.yml]:
```suggestion
                Number of TCP connections in _close_wait_ state
```

File: metricbeat/module/system/socket_summary/_meta/fields.yml [hunk 3]
```diff
@@ -35,6 +35,18 @@
...
+            - name: time_wait
+              type: integer
+              description: >
+                TCP time_wait ports
```
Comment (jsoriano) [inline metricbeat/module/system/socket_summary/_meta/fields.yml]:
```suggestion
                Number of TCP connections in _time_wait_ state
```

**Comment by yaule** [comment]
@jsoriano Thank you!

**Comment by jsoriano** [comment]
jenkins, test this

**Comment by sayden** [comment]
jenkins, test this

**Pull request merged**